### PR TITLE
{data}[foss/2016b,foss/2017a] xmlf90 v1.5.3

### PR DESCRIPTION
**Pull request by pouillon**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/x/xmlf90/xmlf90-1.5.3-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+version = '1.5.3'
+
+homepage = 'https://launchpad.net/xmlf90'
+description = """xmlf90 is a basic XML parsing library written in Fortran."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['https://launchpad.net/xmlf90/trunk/1.5/+download/']
+checksums = ['bf6b8fd83cd1c7fae6f9451a5babde82']
```
Comment (boegel) [inline easybuild/easyconfigs/x/xmlf90/xmlf90-1.5.3-foss-2016b.eb]: please use `SHA256` checksum, avoid hardcoding `1.5`, use `%(version_major_minor)s` instead

Comment (pouillon) [reply easybuild/easyconfigs/x/xmlf90/xmlf90-1.5.3-foss-2016b.eb]: Done and took note for the future.

File: easybuild/easyconfigs/x/xmlf90/xmlf90-1.5.3-foss-2016b.eb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+sanity_check_paths = {
+    'files': ['include/xmlf90_%s.mod' % item for item in ['cml', 'dom', 'sax', 'wxml', 'xpath']] + ['lib/libxmlf90.a'],
+    'dirs': ['include', 'lib'],
```
Comment (pouillon) [inline easybuild/easyconfigs/x/xmlf90/xmlf90-1.5.3-foss-2016b.eb]: Fixed, just as for LibPSML (PR #5859).

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/a6276c8632af72111e059f9941363cab for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ce3f5e42969b32dbf0ab64d926093aea for a full test report.

**Comment by boegel** [comment]
(close/re-open to let Travis re-test, it's been too long...)

**Comment by boegel** [comment]
Going in, thanks @pouillon!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2434.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bbf474e9ea494cc7a66a48540ff41388 for a full test report.

**Pull request merged**